Update howto-manage-backups.md

shaiwuan opened the pull request:
Corrected ibmcloud resource service-instance-create command to include service endpoints.

Thanks for taking the time to contribute to the IBM Cloud docs! After you open your pull request, a maintainer should review it soon.

We don't merge changes directly in this repository because content is not published from here. However, we'll incorporate any changes in our upstream repository so that they're reflected in the docs.

### Description

<!--- Replace this text with a summary of the changes in this pull request. Include why the changes are needed and context about the changes. --->

###  Any other comments?

<!-- Add additional information or screenshots that you think we need.-->

---

### For reviewers (This section is only for IBMers)

You can't merge pull requests here.

- If you can incorporate these changes, copy them to your GitHub Enterprise repo. Leave a comment and close the pull request.
- If you can't accept the changes, leave information about why, and close the pull request.

If you don't have permission to close the request, ask for help in the IBM Cloud \#docs Slack channel.

DailyAlice commented:
Updated in most recent [commit](https://github.com/ibm-cloud-docs/cloud-databases/commit/fc05aa897215646de998c0e9139b0dc46a73b81f). Decided to go with linking to their example, as it is more comprehensive than our existing one.

shaiwuan committed:
Update howto-manage-backups.md

Corrected ibmcloud resource service-instance-create command to include service endpoints.
